walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,68 @@
 
 package com.google.sps;
 
+import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.Collection;
+import java.util.List;
+import java.util.Set;
+import java.util.Timer;
+import java.util.stream.Collectors;
```

Comment:
very minor, but imports should be in alphabetical order as per [this](https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing)